rackcity/admin.py

```diff
@@ -1,6 +1,5 @@
 from django.contrib import admin
 from .models import (
-    AbstractAsset,
```

Comment:
Don't delete pls 🤪

Reply:
oops - this should be AssetID! Django doesn't let you have true abstract classes in admin.py